styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -168,6 +183,30 @@ private BackfillsPayload getBackfills(RequestContext requestContext) {
     return Response.ok();
   }
 
+  private void haltActiveInstances(Backfill backfill, RequestContext rc) throws IOException {
+    final Set<WorkflowInstance> activeWorkflowInstances =
+        storage.readActiveWorkflowInstances(backfill.workflowId().componentId()).keySet();
+
+    final Client client = rc.requestScopedClient();
+
+    rangeOfInstants(backfill.start(), backfill.nextTrigger(), backfill.partitioning()).stream()
+        .map(instant -> WorkflowInstance
+            .create(backfill.workflowId(), toParameter(backfill.partitioning(), instant)))
+        .filter(activeWorkflowInstances::contains)
+        .forEach(workflowInstance -> haltActiveInstance(workflowInstance, client));
+  }
+
+  private void haltActiveInstance(WorkflowInstance workflowInstance, Client client) {
+    try {
+      final ByteString payload = serialize(Event.halt(workflowInstance));
+      Request request = Request.forUri(schedulerApiUrl("events"), "POST")
+          .withPayload(payload);
+      client.send(request);
```

Comment:
This seems to be an asynchronous call and we don't track the outcome of the request? We should notify if some of the `halt` requests failed in the process